src/components/EmojiPicker/index.tsx

```diff
@@ -0,0 +1,71 @@
+import {SFC} from 'types';
+import {useEffect, useRef, useState} from 'react';
+import {mdiEmoticonOutline} from '@mdi/js';
+import Icon from '@mdi/react';
+import EmojiPicker from 'emoji-picker-react';
+import * as S from './Styles';
```

Comment:
Let's try to keep following the convention of sorted/ordered imports.
I know this is not followed everywhere but let's try doing that.
```suggestion
import {useEffect, useRef, useState} from 'react';
import {mdiEmoticonOutline} from '@mdi/js';
import Icon from '@mdi/react';

import EmojiPicker from 'emoji-picker-react';
import {SFC} from 'types';

import * as S from './Styles';
```